### PR TITLE
Feature/ops-668-devstack-default-settings

### PR DESCRIPTION
**Pull request by sydorovdmytro**
**Description:** Default Labster settings for edX Devstack

**JIRA:** https://liv-it.atlassian.net/browse/OPS-668

**Testing instructions:**

1. Run Vagrant box using Labster confluence documentation https://liv-it.atlassian.net/wiki/spaces/UP/pages/12550232/Installing+the+Open+edX+Developer+Stack+in+vagrant+on+developer+machine
2. Check if you see Labster theme
3. Check if you see voucher tab

**Reviewers:**
- [ ] @avkoval 
- [ ] @polesye 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

**Review by TamoshaytisV — APPROVED** [review]

**Comment by polesye** [comment]
@sidorovdmitry why do we change devstack.py? Can we user server-vars.yaml instead? Can we move it to the separate settings file?

**Comment by sydorovdmytro** [comment]
@polesye We can't use server-vars.yml for devstack because we can't use /edx/bin/update command. Why do you want to use separate settings file if devstack.py created for this purpose?

**Comment by polesye** [comment]
@sidorovdmitry 1 line change is better than 49 lines. Less conflicts during upgrades.
We already have https://github.com/Livit/Livit.Learn.EdX/blob/7bd0eefa451996c4dfef70ad6caa834b49fd9f9c/lms/envs/labster.py . Can we re-use it somehow?

One more question: Why local instance points to staging? 
Do we really need to use real emails, so local instance may spam CS?

**Comment by polesye** [comment]
@sidorovdmitry could you explain what are you trying to solve?

**Comment by sydorovdmytro** [comment]
@polesye It was tension from @TamoshaytisV . The main issue - labster specific settings for local development. We can store them in some django settings files (easy and correct way) or put them to the {lms,cms}.env.json files (hard and wrong way). When you run devstack box, you get ready and configured environment and no need to edit json files to set any settings.

ok, I can create separate file but I don't think we can use labster.py for devstack.

> Why local instance points to staging?

Where should we point this environment if we don't have permanent edX sandboxes or local API environment? As for me, by default it can be staging but you can overwrite this settings if you need. We use the same settings for edX sandboxes and don't have any issues.

**Comment by polesye** [comment]
> Where should we point this environment if we don't have permanent edX sandboxes or local API environment? As for me, by default it can be staging but you can overwrite this settings if you need. We use the same settings for edX sandboxes and don't have any issues.

Staging is used for pre-production testing and development instances should NOT point to it to avoid removing any test data or breaking it.
The same about real emails.

Why don't you use? You can easily create it as a part of provision script during box creation?

https://github.com/edx/edx-platform/blob/8435e7b5715dec12ac3c9a147bee4edd083c5dba/lms/envs/devstack.py#L274

File: lms/envs/devstack.py [hunk 1]
```diff
@@ -280,3 +280,52 @@ def should_show_debug_toolbar(_):
...
+
+LABSTER_API_AUTH_TOKEN = ''
+
+LABSTER_API_URL = "https://staging4-region.labster.com/"
+LABSTER_ENDPOINTS = {
+    "available_simulations": "https://staging4-api.labster.com/api/licenses/available_simulations/",
+    "coaches": "https://staging4-region.labster.com/labster/api/coaches",
+    "license_validate": "https://staging4-region.labster.com/api/licenses/validate/",
+    "voucher_activate": "https://staging4-region.labster.com/vouchers/activate/",
+    "voucher_license": "https://staging4-region.labster.com/vouchers/{}/license/"
```
Comment (polesye) [inline lms/envs/devstack.py:314]: shouldn't point to staging.

**Review by polesye — CHANGES_REQUESTED** [review]

**Comment by polesye** [comment]
@sidorovdmitry any updates? If no, please close the PR.

**Pull request closed without merging**